detox/src/utils/exec.js

```diff
@@ -164,22 +163,33 @@ function _joinCommandAndFlags(command, flags) {
   return result;
 }
 
-async function interruptProcess(childProcessPromise, signal = 'SIGINT') {
-  const log = execLogger.child({ fn: 'interruptProcess' });
+const DEFAULT_KILL_SCHEDULE = {
+  SIGINT: 0,
+};
 
+async function interruptProcess(childProcessPromise, schedule) {
```

Comment:
May I suggest - replacing `schedule` with `options`, having `schedule` a specific option inside? I couldn't intuitively and simply find it clear - in the places where this API is used, what the signals map meant.